pkg/gnosis/http.go

```diff
@@ -0,0 +1,58 @@
+package gnosis
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+)
+
+type commonHTTP struct {
+}
+
+func (h *commonHTTP) doPostHttpRequest(ctx context.Context, url string, queryParams map[string]string, bodyParams any) ([]byte, error) {
```

Comment:
Is it possible to add some metrics for this client? For example: https://github.com/goverland-labs/inbox-storage/blob/main/internal/metrics/sdk_transport.go